core/src/Cache/CredentialType.cs

```diff
@@ -29,8 +29,8 @@ namespace Microsoft.Identity.Core.Cache
 {
     internal enum CredentialType
     {
-        IdToken,
-        AccessToken,
-        RefreshToken
+        idtoken,
+        accesstoken,
+        refreshtoken
```

Comment:
Any reason these are not using CamelCasing ?

Reply:
this values are used in Msal schema, were it is lower case

Reply:
Better to have a method that takes in a enum value and gives you the string - it is more intentional.